### ISSUE TITLE
L1 loss increases suddenly

### ISSUE THREAD

**Issue by wh-isper** [junyanz/pytorch-CycleGAN-and-pix2pix#1352]
hello, thanks for your pix2pix model. I am using resnet50(pretrained) as the encoder to construct the Generator, and the result should be better. But while it is training, L1 loss increases suddenly, like this. 
![微信图片_20211215203729](https://user-images.githubusercontent.com/89512441/146188370-770f08a6-8fe1-4eaf-88b9-fe75c553094b.png)
and the final result is worse than pix2pix.
I am wondering what may cause this problem.
Hope for your answers.

**Comment by ninnghazad** [comment]
there can actually be many reasons. to find out i'd try an get some images put out whenever G_L1 is like 2x its running average or something like that, and look at the images for clues. maybe just very different inputs - maybe a rare bug with your normalization or augmentation...

**Comment by wh-isper** [comment]
thank you! I will have a try
